spec/models/invite_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe Invite do
+  it { should belong_to :user }
+  it { should validate_presence_of :email }
+
+  describe "#send_invite" do
+    let(:invite) { Fabricate(:invite, name: "Adam Apple") }
+    before { invite.send_invite }
```

Comment:
Be careful of overusing `before`. It can make your specs hard to read when they grow. Having the setup in each spec is duplication, but it makes your specs to much easier to read. Also, if you change the setup, it doesn't affect all of the spec when you don't want it to.